src/DI/SentryExtension.php

```diff
@@ -43,6 +45,7 @@ public function loadConfiguration()
                 [
                     $this->config[self::PARAM_DSN],
                     $this->config[self::PARAM_ENVIRONMENT],
+                    $this->config[self::PARAM_TRACES_SAMPLE_RATE],
```

Comment:
Having a sample rate in the `register` method doesn't feel right. Register should contain only the bare minimum that's required to use the Sentry. Setting the sample rate should work the same way as other optional settings, through separate setters (see `addSetup` calls below).

- Please add extra setter to the `SentryLogger` just for the traces sample rate.
- Use that instead of `register()`.
- Remove the extra parameter from `register()`.